### DISCUSSION TITLE
blog/design-system/temp

### DISCUSSION THREAD

**Discussion by giscus** [simian114/blog#93] [General]
# blog/design-system/temp

Hello world, it's me!

http://localhost:3000/blog/design-system/temp

**Comment by simian114** [comment]
댓글 기능 테스트

**Reply by simian114** [reply]
giscus 대단한데?

**Comment by hyongti** [comment]
안녕하세요 사남님 ㅎㅎ 현킴입니다 하하하

**Reply by simian114** [reply]
아니 현킴님;; 오랜만입니다 ㅋㅋㅋ 비완성된 블로그를 보여주다니 이거 참 부끄럽군요

잘 지내시나요
